test/k8sT/external_ips.go

```diff
@@ -34,7 +34,12 @@ const (
 	namespaceTest = "external-ips-test"
 )
 
-var _ = Describe("K8sKubeProxyFreeMatrix tests", func() {
+func skipSuite(name string, t func()) bool {
+	return false
+}
+
+// Replace "skipSuite" with "Describe" to enable the suite.
+var _ = skipSuite("K8sKubeProxyFreeMatrix tests", func() {
```

Comment:
What are we gaining by disabling this entire suite? This is testing 396 different test cases in 15 minutes which is roughly 2.7 seconds per test case.

Reply:
20min for each test-net-next run for testing bpf_sock implementation is imho too much. K8sServicesTest already does it. We can run the K8sKubeProxyFreeMatrix suite manually, same as we do e.g. with `-race`.

Reply:
But the `K8sServicesTest` is only a subset of all tests written in `K8sKubeProxyFreeMatrix`

Reply:
Yeah, but `K8sServicesTest` should have the full coverage of bpf_sock.